source/GameData.h

```diff
@@ -127,11 +129,13 @@ class GameData {
 	static const Set<Person> &Persons();
 	static const Set<Phrase> &Phrases();
 	static const Set<Planet> &Planets();
+	static Set<Playlist> &Playlists();
```

Comment:
Why not const?

Reply:
Because I need access to not const functions

Reply:
Idk if thats clean honestly, can't you make the functions const? (or unconst the element you need?)
This is not supposed to be modified by anything outside of the class, and const is the insurance of that